dev-support/docker/Dockerfile

```diff
@@ -62,12 +62,13 @@ ENV LC_ALL en_US.UTF-8
 ###
 # Install grpcio-tools mypy-protobuf for `python3 sdks/python/setup.py sdist` to work
 ###
-RUN pip3 install grpcio-tools mypy-protobuf
+RUN pip3 install --break-system-packages grpcio-tools mypy-protobuf
 
 ###
 # Install useful tools
 # Install distlib to avoid https://github.com/pypa/virtualenv/issues/2006
-RUN pip3 install distlib==0.3.1 yapf==0.29.0 pytest
+# Specify the version of pluggy to fix it to the version installed on the system.
+RUN pip3 install --break-system-packages distlib==0.3.1 yapf==0.29.0 pytest pluggy==1.4.0
```

Comment:
From my dev env, i have this: yapf==0.43.0 distlib==0.3.9 pluggy==1.5.0. Do you need to update them?

Reply:
> yapf==0.43.0 distlib==0.3.9

This seems to work, so I've fixed it.

> pluggy==1.5.0

Unfortunately, this requires 1.4.0 due to the following error.
`ERROR: Cannot uninstall pluggy 1.4.0, RECORD file not found. Hint: The package was installed by debian.`